MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs

```diff
@@ -213,14 +215,30 @@ public class FFProbeVideoInfo
             }
             else
             {
+                // Do not drop existing non-external media streams from the item
                 mediaStreams = new List<MediaStream>();
+                if (currentMediaStreams != null)
+                {
+                    foreach (var mediaStream in currentMediaStreams)
+                    {
+                        if (!mediaStream.IsExternal)
+                        {
+                            mediaStreams.Add(mediaStream);
+                        }
+                    }
+
+                }
                 mediaAttachments = Array.Empty<MediaAttachment>();
                 chapters = Array.Empty<ChapterInfo>();
             }
 
-            await AddExternalSubtitlesAsync(video, mediaStreams, options, cancellationToken).ConfigureAwait(false);
+            // Only fetch and add external media if we have a video stream
+            if (currentMediaStreams != null && currentMediaStreams.Find(i => i.Type == MediaStreamType.Video) != null)
```

Comment:
I don't understand the purpose of this check.  It creates an undesirable situation where external subtitles are not revealed on the first playback attempt.  Reverting this change works better in my tests.